include/openthread/coap.h

```diff
@@ -68,77 +68,77 @@ extern "C" {
  */
 typedef enum otCoapType
 {
-    kCoapTypeConfirmable    = 0x00,  ///< Confirmable
-    kCoapTypeNonConfirmable = 0x10,  ///< Non-confirmable
-    kCoapTypeAcknowledgment = 0x20,  ///< Acknowledgment
-    kCoapTypeReset          = 0x30,  ///< Reset
+    OT_COAP_TYPE_CONFIRMABLE     = 0x00,  ///< Confirmable
+    OT_COAP_TYPE_NON_CONFIRMABLE = 0x10,  ///< Non-confirmable
+    OT_COAP_TYPE_ACKNOWLEDGMENT  = 0x20,  ///< Acknowledgment
+    OT_COAP_TYPE_RESET           = 0x30,  ///< Reset
 } otCoapType;
 
 /**
  * Helper macro to define CoAP Code values.
  *
  */
-#define COAP_CODE(c, d) ((((c) & 0x7) << 5) | ((d) & 0x1f))
+#define OT_COAP_CODE(c, d) ((((c) & 0x7) << 5) | ((d) & 0x1f))
 
 /**
  * CoAP Code values.
  *
  */
 typedef enum otCoapCode
 {
-    kCoapCodeEmpty                  = COAP_CODE(0, 0),  ///< Empty message code
-    kCoapRequestGet                 = COAP_CODE(0, 1),  ///< Get
-    kCoapRequestPost                = COAP_CODE(0, 2),  ///< Post
-    kCoapRequestPut                 = COAP_CODE(0, 3),  ///< Put
-    kCoapRequestDelete              = COAP_CODE(0, 4),  ///< Delete
-
-    kCoapResponseCodeMin            = COAP_CODE(2, 0),  ///< 2.00
-    kCoapResponseCreated            = COAP_CODE(2, 1),  ///< Created
-    kCoapResponseDeleted            = COAP_CODE(2, 2),  ///< Deleted
-    kCoapResponseValid              = COAP_CODE(2, 3),  ///< Valid
-    kCoapResponseChanged            = COAP_CODE(2, 4),  ///< Changed
-    kCoapResponseContent            = COAP_CODE(2, 5),  ///< Content
-
-    kCoapResponseBadRequest         = COAP_CODE(4, 0),  ///< Bad Request
-    kCoapResponseUnauthorized       = COAP_CODE(4, 1),  ///< Unauthorized
-    kCoapResponseBadOption          = COAP_CODE(4, 2),  ///< Bad Option
-    kCoapResponseForbidden          = COAP_CODE(4, 3),  ///< Forbidden
-    kCoapResponseNotFound           = COAP_CODE(4, 4),  ///< Not Found
-    kCoapResponseMethodNotAllowed   = COAP_CODE(4, 5),  ///< Method Not Allowed
-    kCoapResponseNotAcceptable      = COAP_CODE(4, 6),  ///< Not Acceptable
-    kCoapResponsePreconditionFailed = COAP_CODE(4, 12), ///< Precondition Failed
-    kCoapResponseRequestTooLarge    = COAP_CODE(4, 13), ///< Request Entity Too Large
-    kCoapResponseUnsupportedFormat  = COAP_CODE(4, 15), ///< Unsupported Content-Format
-
-    kCoapResponseInternalError      = COAP_CODE(5, 0),  ///< Internal Server Error
-    kCoapResponseNotImplemented     = COAP_CODE(5, 1),  ///< Not Implemented
-    kCoapResponseBadGateway         = COAP_CODE(5, 2),  ///< Bad Gateway
-    kCoapResponseServiceUnavailable = COAP_CODE(5, 3),  ///< Service Unavailable
-    kCoapResponseGatewayTimeout     = COAP_CODE(5, 4),  ///< Gateway Timeout
-    kCoapResponseProxyNotSupported  = COAP_CODE(5, 5),  ///< Proxying Not Supported
+    OT_COAP_CODE_EMPTY                   = OT_COAP_CODE(0, 0),  ///< Empty message code
+    OT_COAP_REQUEST_GET                  = OT_COAP_CODE(0, 1),  ///< Get
```

Comment:
Instead of prefix with `REQUEST` and `RESPONSE`, is it simpler and more consistent to prefix with `CODE`. Request codes are verbs. Response codes are not.